transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/events/UserTypingProvider.kt

```diff
@@ -6,15 +6,17 @@ import com.genesys.cloud.messenger.transport.util.Platform
 import com.genesys.cloud.messenger.transport.util.logs.Log
 import kotlinx.serialization.encodeToString
 
-private const val TYPING_INDICATOR_COOL_DOWN_IN_MILLISECOND = 5000L
-
 internal class UserTypingProvider(
     private val log: Log,
     private val showUserTypingEnabled: () -> Boolean,
     private val getCurrentTimestamp: () -> Long = { Platform().epochMillis() },
 ) {
     private var lastSentUserTypingTimestamp = 0L
 
+    internal companion object {
```

Comment:
That is not really critical to me, but I think that JetBrains guides to use top level declarations for this sort of usage. 
On other hand, I understand the idea behind the using the companion object here. 
Do not have any strong preference in this context.